### ISSUE TITLE
Contacts: new contact form is prefilled

### ISSUE THREAD

**Issue by kotecky** [runbox/runbox7#602]
If a contact card is viewed before the "New contact" button is clicked, its "Basic information" data are pre-filled in the field of the new contact.

**Comment by tadzik** [comment]
Indeed. This is fixed on https://github.com/runbox/runbox7/pull/554, which should be out Soon™

**Comment by castaway** [comment]
#554 is now live, could you recheck this issue please?
